samples/msgext-search/python/aad.manifest.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
Please check and remove not required sections 

**For Example**
<img width="521" height="519" alt="Image" src="https://github.com/user-attachments/assets/14c98b01-d43b-4dc8-b2af-23fe09e2727b" />